qa/vagrant/src/main/java/org/elasticsearch/packaging/test/ArchiveTestCase.java

```diff
@@ -318,4 +318,51 @@ public void test100RepairIndexCliPackaging() {
         }
     }
 
+    public void testAbortWhenJavaHomeContainsQuotes() {
+        assumeThat(installation, is(notNullValue()));
```

Comment:
This should be an assumeTrue for windows. Otherwise the test will show up as "passing" for other distributions, even though it hasn't run anything since all the checks below are within a windows block.